examples/execute-example.bash

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+path=$(find client common -name "*$1*" | grep '\.ts$' | head)
+yarn ts-node $path "${@:2}"
```

Comment:
This is incredibly hacky. Why not tell the user to just run

```
yarn ts-node /path/to/example.ts --account --...
```

These sorts of hacky scripts are really not something we should have.